tools/l10n/check_translations.py

```diff
@@ -37,7 +37,7 @@ def missing_source_exception(lang, code):
 
 def count_mismatch_warning(lang, code):
     # number of same placeholders used in source and target do not match
-    print(" * [{lang}/strings.xml] number of placeholders not matching, key: {code}".format(
+    print(" W [{lang}/strings.xml] number of placeholders not matching, key: {code}".format(
```

Comment:
I think we can be more verbose and actually scream "WARNING: " or "ERROR: " into the face of the script user :)